test/cases/enum.ts

```diff
@@ -20,10 +21,42 @@ export var schema = {
   "additionalProperties": false
 }
 
-export var types = `interface Enum {
+export var configurations = [
+  {
+    settings: {
+      useTypescriptEnums: false
+    },
+    types: `interface Enum {
   foo: "a" | "b" | "c";
   bar: number;
   baz: {
     a: number;
   };
 }`
+  }, 
+  {
+    settings: {
+      useTypescriptEnums: true
+    },
+    types: `enum Foo {
+  a,
+  b,
+  c
+}
+enum Bar {
+  One = 1,
+  Two = 2,
+  Three = 3
+}
+enum Baz {
+  [object Object],
+  [object Object],
+  [object Object]
+}
```

Comment:
Enum members must be JSON schema strings:

> The body of an enum declaration defines zero or more enum members which are the **named values** of the enum type. 
> *https://github.com/Microsoft/TypeScript/blob/master/doc/spec.md#92-enum-members*

We should probably throw a `TypeError('Enum members must be strings when useTypescriptEnums=true')` instead of trying to compile here.